src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,11 @@
-const humanCatDogYears = (number) => {};
+const humanCatDogYears = (number) => {
+  let catYears = 0;
+  let dogYears = 0;
+  if (number > 1) {
+    catYears = 15 + 9 + (number - 2) * 4;
+    dogYears = 15 + 9 + (number - 2) * 5;
+  }
+  return [number, catYears, dogYears];
```

Comment:
This is a nice solution! If you wanted to you could change it a bit to use something called an early exit condition, which is where you account for the easiest solutions at the start of the function and return a value before any calculation needs to be done. In this case, just the case of 0 being passed in would fulfil that, so
```
  if (number === 0) return [0, 0, 0];

  const catYears = 15 + 9 + (number - 2) * 4;
  const dogYears = 15 + 9 + (number - 2) * 5;

  return [number, catYears, dogYears];
```

Again, does the exact same work as before, but is a bit cleaner and more concise. Theres nothing wrong with how you did it, just showing you an alternative approach here is all :)